nss-cohort-36/bangazon-client-illin-illusionists-react
# Sh react

# Description

Show customer and payment info on myprofile page 

Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions

git fetch --all
git checkout sh-react
1. Login as yourself
2. Click on profile from nav bar
3. see that only your info and payment types show up - due to database differences you may not see any payment types
4. in postman, add a payment type 
insert into bangazonAPI_paymenttype 
values(null, null, "American Express", "12334567890", "2019-09-23 14:06:30.047000", "2019-09-23 14:06:30.047000", XXXinput your customer id hereXXXX)
5. verify that your payment type shows up



# Checklist:
